### DISCUSSION TITLE
Resources to keep organized

### DISCUSSION THREAD

**Discussion by bayleeriley** [USU-ITLS-x265-23-summer/class-repo#23] [Q&A] [answered]
Could you let me know what resources you know about organizing this work? I worry about the 4 different resources we are required to use, and how I can keep everything straight.

**Comment by Skye404** [comment, marked as answer]
I could be wrong, but I think FreeCode Camp and Exercism pair with GitHub, so they will link up in GitHub for our assignments. I think the UtahJS Slack group is just another resource for us to use to ask questions and help others. If I understand correctly, there don't seem to be any direct assignments related to the Slack group, it's just a resource we can use.

**Comment by metasean** [comment]
FreeCodeCamp and Exercism are where you're going to be learning and practicing the basics of HTML, CSS, & JavaScript - but make sure you 
- add any notes to the [class-repo](https://github.com/USU-ITLS-x265-23-summer/class-repo)
- post questions, debug requests, etc. in the classroom [discussions](https://github.com/orgs/USU-ITLS-x265-23-summer/discussions)
